cli/tests/integration/js_unit_tests.rs

```diff
@@ -29,17 +131,67 @@ fn js_unit_tests() {
     .arg("--location=http://js-unit-tests/foo/bar")
     .arg("--no-prompt")
     .arg("-A")
-    .arg(util::tests_path().join("unit"))
+    .arg(util::tests_path().join("unit").join(format!("{test}.ts")))
+    .stderr(Stdio::piped())
+    .stdout(Stdio::piped())
     .spawn()
     .expect("failed to spawn script");
 
-  let status = deno.wait().expect("failed to wait for the child process");
+  let now = Instant::now();
+  let stdout = deno.stdout.take().unwrap();
+  let test_name = test.clone();
+  let stdout = std::thread::spawn(move || {
+    let reader = BufReader::new(stdout);
+    for line in reader.lines() {
+      if let Ok(line) = line {
+        println!("[{test_name} {:0>6.2}] {line}", now.elapsed().as_secs_f32());
+      } else {
+        break;
+      }
+    }
+  });
+
+  let now = Instant::now();
+  let stderr = deno.stderr.take().unwrap();
+  let test_name = test.clone();
+  let stderr = std::thread::spawn(move || {
+    let reader = BufReader::new(stderr);
+    for line in reader.lines() {
+      if let Ok(line) = line {
+        eprintln!("[{test_name} {:0>6.2}] {line}", now.elapsed().as_secs_f32());
+      } else {
+        break;
+      }
+    }
+  });
+
+  const PER_TEST_TIMEOUT: Duration = Duration::from_secs(5 * 60);
```

Comment:
Now that I'm thinking about it - for a single file the timeout shouldn't be larger than 1 minute.

Reply:
I feel like 1 minute might be too tight, but I can see if we get flakes.

Reply:
Actually, I just saw on my local machine that the crypto tests take > 60s. Let me try 2 mins.

Reply:
Had to leave this at 5 due to the long crypto tests (even when running in parallel)